lib/inspec/run_data.rb

```diff
@@ -47,7 +47,7 @@ class RunData
     # core reporters have been migrated to plugins. It is probable that new data elements
     # and new Hash compatibility behavior will be added during the core reporter plugin
     # conversion process.
-    SCHEMA_VERSION = "0.1.0".freeze
+    SCHEMA_VERSION = "0.2.0".freeze
```

Comment:
The rundata docs need to be updated - https://github.com/inspec/inspec/blob/94c30647ed8a3374f576379553a73ae7e1587a4c/docs/dev/plugins.md#L449

Reply:
Good call, thanks, I pushed an update to this file.